system/Helpers/inflector_helper.php

```diff
@@ -11,9 +11,7 @@
 
 // --------------------------------------------------------------------
 
-/**
- * CodeIgniter Inflector Helpers
- */
+// CodeIgniter Inflector Helpers
```

Comment:
Sometimes there seems to be an empty line after the comment, sometimes not.
I think there should not be one?

Reply:
We can manually clean these later on.

Reply:
Just a follow up question: If we do manual cleanups, e.g. like in the other comment where you added the 'string' typehint.
Will those not be overwritten by the fixer as soon as someone tries to push changes to this file?

Reply:
Can you clarify further?

Reply:
I was wondering whether running the php-cs-fixer with `phpdoc_add_missing_param_annotation` again would replace your manual

     * @param string $type

with 

     * @param $type

in `Email.php`. I just tested, and it does not, so all good.